posthog/hogql_queries/web_analytics/top_clicks.py

```diff
@@ -0,0 +1,53 @@
+from django.utils.timezone import datetime
+
+from posthog.hogql import ast
+from posthog.hogql.parser import parse_select
+from posthog.hogql.query import execute_hogql_query
+from posthog.hogql_queries.web_analytics.web_analytics_query_runner import WebAnalyticsQueryRunner
+from posthog.hogql_queries.utils.query_date_range import QueryDateRange
+from posthog.models.filters.mixins.utils import cached_property
+from posthog.schema import WebTopClicksQuery, WebTopClicksQueryResponse
+
+
+class TopClicksQueryRunner(WebAnalyticsQueryRunner):
+    query: WebTopClicksQuery
+    query_type = WebTopClicksQuery
```

Comment:
I'd still add `Web` in front of all of these classes to keep it consistent and predictable for future travellers.

Reply:
Fixed